src/components/Identifier/Identifier/Identifier.stories.tsx

```diff
@@ -0,0 +1,426 @@
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="seven">
+    <IdentifierLink href="#">Privacy policy</IdentifierLink>
+  </IdentifierLinkItem>,
+]
+
+const testLinksSpanish = [
+  <IdentifierLinkItem key="one">
+    <IdentifierLink href="#">Acerca de &lt;Parent shortname&gt;</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="two">
+    <IdentifierLink href="#">Soporte de accesibilidad</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="three">
+    <IdentifierLink href="#">Solicitud a través de FOIA</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="four">
+    <IdentifierLink href="#">Datos de la ley No FEAR</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="five">
+    <IdentifierLink href="#">Oficina del Inspector General</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="six">
+    <IdentifierLink href="#">Informes de desempeño</IdentifierLink>
+  </IdentifierLinkItem>,
+  <IdentifierLinkItem key="seven">
+    <IdentifierLink href="#">Política de privacidad</IdentifierLink>
+  </IdentifierLinkItem>,
+]
+
+const testIdentifierGovContent = [
+  <div className="usa-identifier__container" key="one">
+    <div
+      data-testid="identifierGov-description"
+      className="usa-identifier__usagov-description">
+      Looking for U.S. government information and services?
+    </div>
+    &nbsp;
+    <a
+      data-testid="identifierGov-link"
+      href="https://www.usa.gov/"
+      className="usa-link">
+      Visit USA.gov
+    </a>
+  </div>,
+]
+
+const testIdentifierGovContentSpanish = [
+  <div className="usa-identifier__container" key="two">
+    <div
+      data-testid="identifierGov-description"
+      className="usa-identifier__usagov-description">
+      ¿Necesita información y servicios del Gobierno?
+    </div>
+    &nbsp;
+    <a
+      data-testid="identifierGov-link"
+      href="https://www.usa.gov/espanol/"
+      className="usa-link">
+      Visite USAGov en Español
+    </a>
+  </div>,
+]
+
+export const identifierDefault = (): React.ReactElement => (
+  <Identifier>
+    <IdentifierMasthead aria-label="Agency identifier">
+      <div className="usa-identifier__container">
+        <div className="usa-identifier__logos">
+          <IdentifierLogo
+            agencyUrl="#"
+            className="usa-identifier__logo-custom-class-name">
+            {testIdentifierLogo}
+          </IdentifierLogo>
+        </div>
+        <div
+          data-testid="identifierMasthead-agency-description"
+          className="usa-identifier__identity"
+          aria-label="Agency description">
+          <p className="usa-identifier__identity-domain">domain.edu.mil.gov</p>
+          <p className="usa-identifier__identity-disclaimer">
+            {`An official website of the `}
+            <a href="testlink">Test Agency Name</a>
+          </p>
+        </div>
+      </div>
```

Comment:
I like this flexible approach as it does provide value to the consumer, and it takes a complex burden of hardcoding grammar for multiple languages.

There is an enhancement I wanted to surface for reviewers to discuss. I don't think that we necessarily need to do this for _this_ PR, but my inclination would be to create wrapper components for most of this JSX, just to remove the burden on the implementer from needing to interact with the uswds classes. That's one of the main/nice things about our library is that we encapsulate all of that into our components, generally. 

That would involve creating several subcomponents like `IdentifierContainer`, `IdentifierLogos`, `IdentifierAgencyDescription`.

If that is something we agree would be a good idea, I would be happy with creating a Github issue for the enhancement, and merging this as is for now. Then it wouldn't be a breaking change in the future, just a slightly more polished out-of-the-box set of sub-components.

Thoughts?

Reply:
And @SirenaBorracha just to be clear, on this point I'm definitely not expecting you to make any changes for this at the moment. Curious what other reviewers thing as well.

Reply:
@brandonlenz yup I definitely agree!